weapons/forgotten-weapons.yaml

```diff
@@ -630,7 +628,7 @@ GeckoLauncherBarrage:
 ForgottenT3Missile:
 	Inherits: ^MissileAPHE
 	Inherits@wh: ^APHEWH
-	ValidTargets: Ground, Air, MidAir
+	ValidTargets: Ground, Aircraft
```

Comment:
https://github.com/DoGyAUT/OpenRA/pull/17 is supposed to fix that but needs testing.